lambdas/data-migration2/src/granulesAndFiles.ts

```diff
@@ -72,8 +74,13 @@ const initializeGranulesAndFilesMigrationResult = (): GranulesAndFilesMigrationR
  */
 export const migrateGranuleRecord = async (
   record: { [key: string]: NativeAttributeValue },
-  trx: Knex.Transaction
+  trx: Knex.Transaction,
+  migrationParams: {
+    migrateAndOverwrite: string,
```

Comment:
The PR adds updated migration behavior flags to the migration params.  Most of these changes are just pass-through on the param options.